server/market/market.go

```diff
@@ -554,14 +575,9 @@ func (m *Market) Run(ctx context.Context) {
 		close(notifyChan)
 		wgFeeds.Wait()
 
-		// Close and delete the outgoing order feed channels as a signal to
-		// subscribers that the Market has terminated.
-		m.orderFeedMtx.Lock()
-		for _, s := range m.orderFeeds {
-			close(s)
-		}
-		m.orderFeeds = nil
-		m.orderFeedMtx.Unlock()
+		// Retain the outgoing order feed channels, which are the links to the
+		// book router and any other consumers, for possible Market resume, and
+		// for Swapper's unbook callback to function using sendToFeeds.
```

Comment:
Closing these channels was only a trick to get the book router's msgBook goroutine to auto-stop when the market is was receiving from stopped.  The book router stopping via context cancellation is more appropriate anyway.